index.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: docs
+title: "Rockefeller Archive Center Digital Media Transfer Workflow"
+---
+
+## Digital Media Transfer Overview
+
+This is the Rockefeller Archive Center's documentation for digital media inventorying and transfer workflows using the Digital Media Log.
```

Comment:
Again, need to define "digital media".

Reply:
hi @a-berish  @kemartin2015 , could both of you look over the definitions I put into the readme and index pages for digital media and the other text I added in those sections? please feel free to add to them or change them